Shared/Samples/Add 3D tiles layer/Add3DTilesLayerView.swift

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct Add3DTilesLayerView: View {
+    /// A scene with dark gray basemap and an OGC 3D tiles layer.
+    @State private var scene: ArcGIS.Scene = {
+        // Creates a scene and sets an initial viewpoint.
```

Comment:
```suggestion
        // Create a scene and sets an initial viewpoint.
```

Do all the comments in this closure need to be updated like this?

Reply:
@yo1995 Should all comments in the Sample Viewer be 3rd person singular? I was under the impression that only only the doc ones where. I have been doing that since that seemed like the pattern in the other samples. If so, should all the non 3rd person singular comments be updated?

Reply:
See the "use-3rd-person-singular-in-comments-" section in the guide. No rules apply to //, me personally tend to keep them consistent.